backend/blueprint/api/controllers/registerUser.py

```diff
@@ -14,20 +14,15 @@ def registerUser():
             Response(500).generateMessage(
                 str(e)))
 
-    newUser = User(
-        data['username'],
-        data['password'],
-        data['name'],
-        data['email'])
-
     try:
-        newUser.save_to_db()
+        auth.create_user(
+            display_name=data['name'],
+            email=data['email'],
+            phone_number=data['phone_number'],
```

Comment:
Don't copy directly from the documentation. Please see the use cases that are currently present in the system. Is there `photo_url` or `phone_number` present. If not them first add and then update. Please read documentation for reference don't just copy paste the code.

Reply:
It is also returning a `userRecord` which can be of use later in future so store the necessary details in db. Ref [Link](https://firebase.google.com/docs/reference/admin/python/firebase_admin.auth#create_user)

Reply:
@vaibhavsingh97 please send a PR that has complete work done. And not partial fixes for small tasks.
Moreover please do not copy and paste from docs as Yash pointed. You should atleast look into what you're pasting and if it is even relevant to the context of the project.